app/views/users/index.html.erb

```diff
@@ -7,7 +7,7 @@
     </div>
     <div class="lg-12-3">
       <% unless current_user %>
-        <%= link_to user_omniauth_authorize_path(:github), class: "btn btn-github" do %>
```

Comment:
Merhaba @msdundar geciktiğim için kusura bakma lütfen.

Local'imde `user_github_omniauth_authorize_path` diye bir route bulunamadı diyor, hatta senin branch'indeyken değiştirdiğimde de aynı hatayı alıyorum. `user_omniauth_authorize_path(:github)` ile düzgünce login olabiliyorum.

Github authorization için ise `config/initializers/devise.rb` dosyasındaki `config.omniauth` kısmında client id ve secret giriyoruz. Production'da bunlar ENV'den alınıyor ama development'da silip hardcode edebilirsiniz. Bu değerleri almak için ise sadece yeni Github app'i açmak yeterli: https://github.com/settings/applications/new